packages/rocketchat-cloud/client/admin/cloud.html

```diff
@@ -0,0 +1,83 @@
+<template name="cloud">
+	<div class="main-content-flex">
+		<section class="page-container page-home page-static page-settings">
+			{{> header sectionName="Rocket.Chat Cloud Connect"}}
+			<div class="content">
+				{{#requiresPermission 'manage-cloud'}}
+					<div class="section">
+						<div class="section-title">
+							<div class="section-title-text">
+								What is this?
+							</div>
+						</div>
+
+						<div class="section-content">
+							<p>Rocket.Chat Cloud Connect allows you to connect your self-hosted Rocket.Chat Workspace to our Cloud. Doing so enables you to manage your licenses, Billing and Support in Rocket.Chat Cloud.</p>
+						</div>
+					</div>
+					<div class="section">
+						{{#if info.registeredWithWizard}}
+							{{#if info.workspaceConnected}}
+								<div class="section-content border-component-color">
+									{{#if info.userAssociated}}
+										<p>Your workspace is now connected to the Rocket.Chat Cloud and an account is associated.</p>
+									{{else}}
+										<p>Your workspace is now connected to the Rocket.Chat Cloud. If you would like, you can login to the Rocket.Chat Cloud and associate your workspace with your Cloud account.</p>
+										<div class="input-line double-col">
+											<label class="setting-label" title=""></label>
+											<div class="setting-field">
+												<button type="button" class="rc-button rc-button--primary action login-btn">Login To Rocket.Chat Cloud</button>
+											</div>
+										</div>
+									{{/if}}
+								</div>
+							{{else}}
+								<div class="section-content border-component-color">
+									<div class="input-line double-col">
+										<label class="setting-label" title="cloudEmail">Email</label>
+										<div class="setting-field">
+											<input class="input-monitor rc-input__element" type="text" name="cloudEmail" value="{{ info.email }}">
```

Comment:
even if I have added my email on wizard's first step field "Organization Email" this field was blank, is that correct?

Reply:
That is odd. In all of my testing that never happened to me.

Reply:
This never happened to me either.  🤔 Had you already filled the wizard out prior to checking this Branch out?  Just trying to figure out how to reproduce

Reply:
maybe the issue was that check on the `Organization_Email` setting.. on current state it is working as expected :+1: